### PR TITLE
A slider

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: ksp_plugin_adapter/differential_slider.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                            double log10_lower_rate,
+                            double log10_upper_rate)
+      : this(label,
+             unit,
+             log10_lower_rate,
+             log10_upper_rate,
+             -double.NegativeInfinity,
+             -double.PositiveInfinity) {}
+
+  // Rates are in unite of |value| per real-time second.
```
Comment (pleroy) [inline ksp_plugin_adapter/differential_slider.cs]: unite?  Also, there is no |value|.

File: ksp_plugin_adapter/differential_slider.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    unit_ = unit;
+    culture_= new CultureInfo("");
+    culture_.NumberFormat.NumberGroupSeparator = "'";
+    log10_lower_rate_ = log10_lower_rate;
+    log10_upper_rate_ = log10_upper_rate;
+    min_value_ = min_value;
+    max_value_ = max_value;
+  }
+
+  // Returns true if and only if |value| changed.
```
Comment (pleroy) [inline ksp_plugin_adapter/differential_slider.cs]: Maybe explain what it does.

File: ksp_plugin_adapter/differential_slider.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        rightValue : 1,
+        options    : UnityEngine.GUILayout.ExpandWidth(true));
+    bool value_changed = false;
+
+    if (UnityEngine.GUILayout.Button("0", UnityEngine.GUILayout.Width(20))) {
+      value_changed = true;
+      value = 0;
+    }
+    double log10_lower_rate = -3;
+    double log10_upper_rate = 3.5;
```
Comment (pleroy) [inline ksp_plugin_adapter/differential_slider.cs]: Shouldn't this use the members?

File: ksp_plugin_adapter/differential_slider.cs [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  // Returns true if and only if |value| changed.
+  public bool Render() {
+    UnityEngine.GUI.skin = null;
+    UnityEngine.GUILayout.BeginHorizontal();
+
+    UnityEngine.GUILayout.Label(
+        text       : label_,
+        options    : UnityEngine.GUILayout.Width(100));
```
Comment (pleroy) [inline ksp_plugin_adapter/differential_slider.cs]: Could we have constants for 100 (here and at line 63), 50 (line 67), 21 (lines 71-72) and 20 (line 80)?

Comment (eggrobin) [inline ksp_plugin_adapter/differential_slider.cs]: They're all pretty much ad hoc.

**Pull request merged**